Crashlytics/UnitTests/Mocks/FIRCLSTempMockFileManager.h

```diff
@@ -1,48 +0,0 @@
-// Copyright 2019 Google
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#import "Crashlytics/Crashlytics/Models/FIRCLSFileManager.h"
-
-#import <XCTest/XCTest.h>
-
-@interface FIRCLSTempMockFileManager : FIRCLSFileManager
```

Comment:
I'm curious about the reason behind removing the TempMockFileManager. Were you happening to see flakes in the tests before this refactor?